srcs/runners_manager/vm_creation/github_actions_api.py

```diff
@@ -26,9 +26,19 @@ def link_download_runner(self, archi='x64'):
             if elem['os'] == 'linux' and elem['architecture'] == archi
         )
 
-    def get_runners(self):
+    def _get_runner_page(self, page=1, per_page=30):
         info_link = f'https://api.github.com/orgs/{self.organization}/actions/runners'
-        return self.session.get(info_link).json()
+        return self.session.get(info_link, params={'page': page, 'per_page': per_page}).json()
+
+    def get_runners(self):
+        index = 1
+        page = self._get_runner_page(page=index, per_page=100)
+
+        while index * 100 < page['total_count']:
+            index += 1
+            page['runners'] += self._get_runner_page(page=index, per_page=100)['runners']
+
+        return page
```

Comment:
You are using 3 times the hardcoded value 100. It could be defined in a var at the beginning of your function.